modules/API/API.js

```diff
@@ -1770,6 +1770,24 @@ class API {
         });
     }
 
+    /**
+     * Notify the external application that the audio or video is being shared by a participant.
+     *
+     * @param {string} mediaType - Whether the content which is being shared is audio or video.
+     * @param {string} value - Whether the sharing is playing, pause or stop (on audio there is only playing and stop).
+     * @param {string} participantId - Participant id of the participant which started or ended
+     *  the video or audio sharing.
+     * @returns {void}
+     */
+    notifyAudioOrVideoSharingToggled(mediaType, value, participantId) {
+        this._sendEvent({
+            name: 'audio-or-video-sharing-toggled',
```

Comment:
Considering these are two separate features, shouldn't we have two separate events? ()one for audio and another for video

Reply:
This was agreed with Saul when I started the implementation, only one event with different type for audio/video.